### ISSUE TITLE
Using a hotkey to create a new sequence when out of them should show a dialog box

### ISSUE THREAD

**Issue by Chordian** [Chordian/sidfactory2#82]
When using **Ctrl+F**, **Ctrl+Shift+F** or **Ctrl+D** and the editor hits the sequence cap (i.e. 7F), a dialog box error should pop up. It could say something in the line of this:

`Cannot create the new sequence. All of the sequences have been used up. Try optimizing your song to free up more sequences.`

**Comment by Chordian** [comment]
See also issue #81 about a related wrap bug.

**Comment by micheldebree** [comment]
Personally I wouldn't mind if it just stops at 7f without a dialog popping up. It would be obvious that I reached the maximum. Pop-ups that I have to click away annoy me a bit when I am in a flow. Then again, this would probably almost never happen. Is there maybe a more subtle way to give info without having to click away a pop-up? Like in a status line somewhere?

**Comment by Chordian** [comment]
Sure, a "Maximum sequence reached" could briefly flash in the top status line. I have thought of the same for quick saving with a "Saved" so perhaps a general timed message handler could be added for these types of messages.

**Comment by micheldebree** [comment]
Yeah good one, elegant solution to the quicksave dialog discussion we had earlier :)
